types/opentype.js/index.d.ts

```diff
@@ -167,48 +167,48 @@ export interface Field {
 
 export class Glyph {
     index: number;
-    private xMin;
-    private xMax;
-    private yMin;
-    private yMax;
-    private points;
-
-    name: string;
-    path: Path | (() => Path);
-    unicode: number;
+    name: string | null;
+    unicode?: number | undefined;
     unicodes: number[];
-    advanceWidth: number;
+    xMin?: number | undefined;
+    xMax?: number | undefined;
+    yMin?: number | undefined;
+    yMax?: number | undefined;
+    advanceWidth?: number | undefined;
+    leftSideBearing?: number | undefined;
+    path: Path;
```

Comment:
The getter always returns a `Path`, while the setter also accepts `() => Path`, but I think it's more practical to not include it in the type. It's not possible to type them separately right now, unfortunately: https://github.com/microsoft/TypeScript/issues/43662